README.md

```diff
@@ -1,38 +1,64 @@
 # Slice for Visual Studio Code
 
-This language extension adds support for highlighting Slice files within Visual Studio Code.
-It supports highlighting the following file types:
+This extension provides support for the Slice Interface Definition Language (IDL).
+
+## Features
+
+- Syntax highlighting
+- Syntax validation
+- Error detection & reporting
+- 'Go to Definition' jumping
+
+### Syntax Highlighting and Validation
+
+This extension supports syntax highlighting and validation for the following Slice file types:
 
 - `.ice` files, which use the [original Slice](https://doc.zeroc.com/ice/latest/the-slice-language) syntax
 - `.slice` files, which use the [latest Slice](https://docs.icerpc.dev/slice2) syntax
 
-Additionally, this extension provides a Slice Language Server, for `.slice` files only.
+### Error Detection and Reporting
+
+Error checking is triggered every time a Slice file is saved or opened,
```

Comment:
Why is this line so short?

Reply:
I don't know, but I put it all on one line.